static/app/views/settings/organizationDeveloperSettings/sentryFunctionDetails.tsx

```diff
@@ -60,6 +59,20 @@ export default function SentryFunctionDetails(props: Props) {
     addSuccessMessage(t('Sentry Function successfully saved.', data.name));
   };
 
+  // TODO: Should type the editor better, don't use any
+  const editorRef: MutableRefObject<null | any> = useRef(null);
+
+  function handleEditorDidMount(editor) {
+    editorRef.current = editor;
+  }
+  function updateCode() {
+    if (editorRef.current) {
+      form.current.setValue('code', editorRef.current.getValue());
+    }
+  }
+
+  const defaultCode =
+    "exports.yourFunction = (req, res) => {\n\tlet message = req.query.message || req.body.message || 'Hello World!';\n\tconsole.log('Query: ' + req.query);\n\tconsole.log('Body: ' + req.body);\n\tres.status(200).send(message);\n};";
```

Comment:
Can you use the backtick string here instead? So that this will actually make sense to read